ask-sdk-core/tests/unit/test_utils.py

```diff
@@ -644,3 +647,19 @@ def test_is_new_session(self):
             self.test_new_session,
             "is_new_session method returned incorrect session information "
             "from input request when a session exists")
+
+    def test_get_user_id_with_no_user_info_returns_none(self):
+        test_input = self._create_handler_input(
+            request=self.test_launch_request)
+        test_input.request_envelope.context.system.user = None
+        self.assertEqual(
```

Comment:
Since this is a None check, probably [assertIsNone](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIsNone) is the right call.

Reply:
Thanks @nikhilym, I have made the changes in the new commit.